xml/admin-keystone.xml

```diff
@@ -1406,14 +1406,31 @@ domain_name_url_safe = off</screen>
                     value as the root user:</para>
```

Comment:
Is there a section where it takes you thru the barclamp configuration? The edits that are made to the barclamp configuration require edits to the json file in raw mode. 
>
   "domain_specific_config": {
    "ldap_users": {
      "identity": {
        "driver": "ldap"
      },
      "ldap": {
        "url": "ldaps://ldap.suse.de",
        "user": "dc=Manager,dc=example,dc=com",
        "password": "",
        "suffix": "dc=suse,dc=de",
        "page_size": 0,
        "alias_dereferencing": "default",
        "query_scope": "one",
        "user_tree_dn": "ou=accounts,dc=suse,dc=de",
        "user_filter": "",
        "user_objectclass": "posixAccount", 
`